ios/Firestack/FirestackRemoteConfig.m

```diff
@@ -0,0 +1,232 @@
+//
+//  FirestackRemoteConfig.m
+//  Firestack
```

Comment:
@Salakar @Ehesp @chrisbianca  Hello guys, please can you help me with objective-c code.